Makefile

```diff
@@ -119,36 +119,45 @@ init: restart
 	$(MAKE) build
```

Comment:
The diff's better this time--this is pretty accurate as to what I changed and where